### ISSUE TITLE
Don't know how to read property "TAGame.GameEvent_TA:bCanVoteToForfeit"

### ISSUE THREAD

**Issue by danielsamuels** [tfausak/octane#64]
https://media.rocketleaguereplays.com/uploads/replay_files/000D0011001F00090007036C5166B13A.replay

```
octane-0.16.2-osx: Data.Binary.Get.runGet at position 422: Don't know how to read property "TAGame.GameEvent_TA:bCanVoteToForfeit"
CallStack (from HasCallStack):
  error, called at libraries/binary/src/Data/Binary/Get.hs:342:5 in binary-0.8.3.0:Data.Binary.Get
```

Interestingly, this replay doesn't have a valid UUID4 ID: `000D0011-001F-0009-0007-036C5166B13A`

**Comment by danielsamuels** [comment]
Just noticed that you fixed this a couple of days ago in cdf4ab87977ea4767b327ea780fb857692b34837. Do you know when you plan to make a new release?

**Comment by tfausak** [comment]
Thanks for the reminder! I just released [0.16.3](https://github.com/tfausak/octane/releases/tag/0.16.3). The binaries should be available in less than an hour.
